modules/ossm-vs-istio.adoc

```diff
@@ -27,7 +27,7 @@ The command line tool for {SMProductName} is `oc`.  {SMProductName} does not s
 
 The upstream Istio community installation automatically injects the sidecar into pods within the projects you have labeled.
 
-{SMProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` annotation as described in the Automatic sidecar injection section.
+{SMProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` label, or annotation, as described in the Automatic sidecar injection section.
```

Comment:
Suggestions (mostly subjective nitpicks):
- `but requires you to opt in to injection` --> "but you must opt in to injection"
- `To enable automatic injection you specify` --> "To enable automatic injection, specify"
- `described in the Automatic sidecar injection section` --> Although I am aware about the OpenShift xref guidelines, from a reader's perspective, a link to that section would be nice.

As a whole, LGTM.

Reply:
Thanks for the style suggestion, @sounix000 I've made the updates; however, I am not able to link across repos.

Reply:
Made style updates; unable to link across repos.